testutils/db.go

```diff
@@ -97,3 +98,16 @@ func TruncateTables(t testing.TB, db *gorm.DB, models []interface{}) {
 		require.NoError(t, err)
 	}
 }
+
+func GetLocalDB(cfg configuration.API) (*gorm.DB, error) {
```

Comment:
I didn't find usage. maybe that is the extra function

Reply:
Usage is still in draft, removed until be ready